src/mlpack/methods/ann/activation_functions/elliot_function.hpp

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file elliot_function.hpp
+ * @author Bisakh Mondal
+ *
+ * Definition and implementation of the Elliot Activation function as described 
+ * by D.L. Elliott
+ *
+ * For more information, see the following paper.
+ *
+ * @code
+ * @techreport{elliott1993better,
+ *   title = {A better activation function for artificial neural networks},
+ *   author = {Elliott, David L},
+ *   url = {https://drum.lib.umd.edu/bitstream/handle/1903/5355/TR\_93-8.pdf}
+ *   year = {1993}
+ * }
+ * @endcode
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_ELLIOT_FUNCTION_HPP
+#define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_ELLIOT_FUNCTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The Elliot function, defined by
+ *
+ * @f{eqnarray*}{
+ *  f(x) &=& \frac{x}{1 + |x|} \\
+ *  f'(x) &=& \frac{1}{(1 + |x|)^2}
+ * @f}
+ */
+
```

Comment:
Hi @bisakhmondal, Could you please remove this empty line from here. I don't think it is required. Thanks.